modules/33acrossBidAdapter.js

```diff
@@ -137,62 +137,82 @@ function _validateVideo(bid) {
 // NOTE: With regards to gdrp consent data, the server will independently
 // infer the gdpr applicability therefore, setting the default value to false
 function buildRequests(bidRequests, bidderRequest) {
+  const ttxSettings = config.getConfig('ttxSettings');
   const gdprConsent = Object.assign({
     consentString: undefined,
     gdprApplies: false
   }, bidderRequest && bidderRequest.gdprConsent);
-
   const uspConsent = bidderRequest && bidderRequest.uspConsent;
   const pageUrl = (bidderRequest && bidderRequest.refererInfo) ? (bidderRequest.refererInfo.referer) : (undefined);
 
   adapterState.uniqueSiteIds = bidRequests.map(req => req.params.siteId).filter(utils.uniques);
 
-  return bidRequests.map(bidRequest => _createServerRequest(
-    {
-      bidRequest,
-      gdprConsent,
-      uspConsent,
-      pageUrl
-    })
-  );
+  const bidRequestsComplete = bidRequests.map(_inferProduct);
+
+  const enableSRAMode = ttxSettings && ttxSettings.enableSRAMode;
+
+  const keyFunc = (enableSRAMode === 'true') ? _getSRAKey : _getDefaultKey;
```

Comment:
this config is a string?? does prebidjs not support boolean?

Reply:
It should. Just made a string for no reason. I can change to bool. Also I wonder if we should use https://docs.prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setBidderConfig instead of `setConfig`

Reply:
i agree to switch to the bidderConfig, thats more fit with the prebidjs api doc